templates/simple_register_lookup.html

```diff
@@ -73,5 +73,51 @@ <h3>Embed this table</h3>
         <script src="/static/js/simple_register_lookup_core.js" type="text/javascript"></script>
         <script src="/static/js/simple_register_lookup.js" type="text/javascript"></script>
         <script src="/static/js/test_site.js" type="text/javascript"></script>
+         <script type="text/javascript">
+        $( document ).ready(function() {
+ 
+        var urlParams = new URLSearchParams(window.location.search);
+        var Device = urlParams.get('Device'); 
+        var DevicesTag = urlParams.get('Tag'); 
+        var TagsExpand = urlParams.get('Expand'); 
+        var deviceDropdown = document.getElementById('device-dropdown');
+        var len = deviceDropdown.options.length;
+        for(i = 0; i < len; i++)
+         {
+         if (deviceDropdown.options[i].innerHTML == Device)
```

Comment:
Please use === only, since == has weird / hard-to-remember rules.

Reply:
Good catch I forgot about that, Done